src/test/java/com/tngtech/java/junit/dataprovider/internal/DataConverterTest.java

```diff
@@ -323,8 +315,7 @@ public void testConvertShouldReturnOneElementForStringArrayWithOneElementSplitBy
         List<Object[]> result = underTest.convert(data, false, parameterTypes, dataProvider);
 
         // Then:
-        assertThat(result).hasSize(1);
-        assertThat(result.get(0)).isEqualTo(new Object[] { "foo", true });
+        assertThat(result).containsExactly(new Object[] { "foo", true });
```

Comment:
maybe we cloud use a helper for creating the array instead of using ```new Object[] { ... }``` over and over.
Do you also think that it would be better for the test cases above to write down the data explicitly instead of reusing the test data somehow?